h/views/api/group_members.py

```diff
@@ -15,8 +21,10 @@
     permission=Permission.Group.READ,
 )
 def list_members(context: GroupContext, _request):
-    """Return a list of a group's members."""
-    return [UserJSONPresenter(user).asdict() for user in context.group.members]
+    return [
+        GroupMembershipJSONPresenter(membership).asdict()
+        for membership in context.group.memberships
+    ]
```

Comment:
The get-group-members API now uses the new `GroupMembershipJSONPresenter` instead of `UserJSONPresenter`. It returns the same dicts as `UserJSONPresenter` does but with `roles` added. This means that this PR also adds `roles` to the get-group-members API responses.